projects/packages/forms/src/dashboard/integrations/google-sheets-card.tsx

```diff
@@ -89,6 +89,16 @@ const GoogleSheetsDashboardCard = ( {
 						<Button variant="link" onClick={ handleViewResponsesClick }>
 							{ __( 'View Form Responses', 'jetpack-forms' ) }
 						</Button>
+						<span>|</span>
```

Comment:
Would it make sense to implement this as a `HStack` as well? So that there is consistence between the other implementation.

Reply:
PS. In response to other comment, I'll also do a follow up that adds some tracking.